src/backend/distributed/cdc/Makefile

```diff
@@ -3,14 +3,21 @@ ifndef DECODER
 endif
 
 MODULE_big = citus_$(DECODER)
-citus_subdir = src/backend/distributed/cdc
-citus_top_builddir = ../../../..
+
 citus_decoders_dir = $(DESTDIR)$(pkglibdir)/citus_decoders
 
+# Set the default value to citus_top_builddir as relative path to the
+# citus top directory from current directory.
+ifeq ($(strip $(citus_top_builddir)),)
+citus_top_builddir = ../../../..
+citus_subdir = $(shell pwd)
```

Comment:
would this work if I were doing `make -C src/backend/distributed/cdc`?

Reply:
Chaged it to absolute path now.